guides/V1_ActionsEvents.md

```diff
@@ -0,0 +1,199 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'actions-and-events'
+is: 'experimental'
+---
+## Actions and Events
+
+Watson Work Services gives your App two patterns for dealing with events, which take place either due to changes in the data model and/or due to actions humans take using a participating client in Watson Work Services such as the Workspace clients. 
+
+Your App can Listen to Events, through secure web hooks. Also, your App can handle actions that may be invoked 
```

Comment:
Remove this paragraph.  It is superfluous with the above one.  Also, in the future a user entered command could trigger a UI based action vs a webhook...